cloud/disk_manager/internal/pkg/dataplane/snapshot/storage/schema/schema.go

```diff
@@ -50,6 +50,27 @@ func Create(
 	}
 	logging.Info(ctx, "Created deleting table")
 
+	err = db.CreateOrAlterTable(
+		ctx,
+		config.GetStorageFolder(),
+		"incremental",
+		persistence.NewCreateTableDescription(
+			persistence.WithColumn("zone_id", persistence.Optional(persistence.TypeUTF8)),
+			persistence.WithColumn("disk_id", persistence.Optional(persistence.TypeUTF8)),
+			persistence.WithColumn("snapshot_id", persistence.Optional(persistence.TypeUTF8)),
+			persistence.WithColumn("checkpoint_id", persistence.Optional(persistence.TypeUTF8)),
+			// Even if created_at timestamp is incorrect then we will just copy
```

Comment:
```
// We use timestamp to make it easier to find the latest snapshot, and It is OK to depend on time here, because clock discrepancy won't hurt the correctness, it can only increase amount of data to be copied but the probability of such event is very low, that's why it can be neglected.
```